### PR TITLE
Euks taxonomy and quality processes

### PR DESCRIPTION
**Pull request by Ales-ibt**

### CODE REVIEW

File: bin/bat_taxo_process.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                    concat_list.append(genome)
+                    classification = line.pop(0)
+                    concat_list.append(classification)
+                    reason = line.pop(0)
+                    concat_list.append(reason)
+                    lineage = line.pop(0)
+                    concat_list.append(lineage)
+                    lineage_scores = line.pop(0)
+                    concat_list.append(lineage_scores)
+                    to_concat.write("\t".join(concat_list)+'\n')
```
Comment (KeteSakharova) [inline bin/bat_taxo_process.py:42]: To be honest, I don't understand why do you use pop(0) five times. Did you need just first 5 values from `line`
would `concat_list = line[:5]` work?

**Review by KeteSakharova — CHANGES_REQUESTED** [review]
Thank you so much, Ales!
I have a couple of python implementation questions :)

File: bin/bat_taxo_process.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    parser.add_argument(
+        "--output", 
+        help="path for the output table", 
+        default="euk_taxonomy.csv", 
+        type=str
+    )
+    parser.add_argument(
+        "--bat_names", 
+        help="BAT names files (*.BAT_run.bin2classification.names.txt)", 
+        nargs="*",
```
[the remark this answers is not in the document]
Comment (Ales-ibt) [reply bin/bat_taxo_process.py:64]: Good catch!

File: bin/bat_taxo_process.py (hunk 1 above)
Comment (Ales-ibt) [reply bin/bat_taxo_process.py:42]: Thank you for your comment, the code looks so much cleaner now.

**Review by KeteSakharova — APPROVED** [review]
Thank you, Ales! Brilliant!

**Pull request merged**